### DISCUSSION TITLE
2Q - Editing my username in GitHub Community, Deleting Old Posts

### DISCUSSION THREAD

**Discussion by indigofficial** [community/community#22220] [New to GitHub]
<p>Hello! I have two questions:</p>
<p><strong>Q1.</strong> I’d like to edit my name in here, but when I click on my Profile Picture &gt; Preferences &gt; Name, I don’t find an input field to change my name.</p>
<p><strong>Q2.</strong> If possible, could mod please delete <em>all</em> my old posts? (Including questions, answers, comments, etc.)</p>
<p>Thank you :smiley:</p>

**Comment by indigofficial** [comment]
<p>What I mean by deleting all my questions, answers, comments, etc. is just to delete <em>all</em> of my questions, comments, and answers, except for this question…<em>if</em> it’s possible, that is.</p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hi <a class="mention" href="/cookibytes">@cookibytes</a>,</p>
<p>Thank you for the request. We’ve removed the older topics.  You should be able to edit the “name” field in the community forum by editing Settings &gt; Profile &gt; Name in your <a href="http://github.com">github.com</a> account. You’ll want to sign out and back into the community for the SSO process to refresh the name field on the community forum after making that change in <a href="http://github.com">github.com</a>.</p>
<p>Let us know if that helps!</p>

**Comment by indigofficial** [comment]
<p>Hi <a class="mention" href="/nyahbhinghiprincess">@nyahbhinghiprincess</a>, thanks, my username thing is fixed, and thanks for closing my posts. However, is there a way to <em>hide</em> my posts? Or delete it from being viewed?</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Hi <a class="mention" href="/cookibytes">@cookibytes</a>, that’s great to hear. Each of the topics that you flagged are also <a href="https://meta.discourse.org/t/what-is-the-difference-between-closed-unlisted-and-archived-topics/51238">unlisted</a> and hidden.</p>

**Comment by indigofficial** [comment]
<p>Hi <a class="mention" href="/nyahbhinghiprincess">@nyahbhinghiprincess</a> how come I can still see it though? Also is there a way where I can edit my old posts?  I can’t see a pencil icon / button anywhere…but I can see it on my more recent posts.</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Hi <a class="mention" href="/cookibytes">@cookibytes</a> you will still be able to view the history when logged in. Do you see the same listing under <strong>Topics</strong> for your profile when viewing it logged out? I can also open the previous topics, so that you can edit them.</p>

**Comment by indigofficial** [comment]
<p>Hi <a class="mention" href="/nyahbhinghiprincess">@nyahbhinghiprincess</a>, I can still see some when logged out Also, you don’t need to open the previous topics, but thank you a lot for your help :smiley:</p>
<p>I think that should be it, most of the topics are gone I think. Thank you once again!</p>
